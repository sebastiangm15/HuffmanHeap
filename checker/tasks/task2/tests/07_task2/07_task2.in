25
43 SATLFGV
50 SATLOMO
24 SATL8CO
18 SATL447
13 SATL1KL
39 SATL60L
46 SATLN3M
31 SATLZRF
5 SATLYSR
12 SATLSZ8
16 SATLTZE
19 SATLW8S
16 SATL4YS
16 SATL7N8
45 SATLOXL
41 SATLYOA
37 SATLO9O
7 SATL46D
27 SATLRPO
22 SATLRLW
42 SATL0U8
19 SATLUKA
37 SATL6OW
14 SATL133
33 SATLEXS
10
10010001010000110110
0110011111010111110011000
101000111011101100011111010001
11001011100100011001100110011011001
0010000010110101111011010101011111111101
110001111010100110001010010010100110111110010
11001000100100001001101111000001111000011101001011
0010111101101110001110011011010011100101010010111100001
010010001011100000011000100100100110100011001000110010110111
00001001100011101101110000010000111101110101001110010011011101101
