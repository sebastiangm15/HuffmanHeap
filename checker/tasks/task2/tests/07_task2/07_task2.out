SATLFGV SATLEXS SATLO9O SATL6OW SATL60L 
SATL60L SATLYOA SATLOMO SATLYOA SATL8CO SATL0U8 
SATLOXL SATL6OW SATLN3M SATLN3M SATLEXS SATLRPO SATL0U8 
SATL8CO SATLYOA SATLTZE SATL60L SATL60L SATL60L SATL60L SATL8CO 
SATLTZE SATLEXS SATL60L SATLN3M SATLOMO SATLOXL SATLOXL SATLZRF SATLRPO 
SATLRLW SATLRPO SATLOXL SATL60L SATL447 SATLTZE SATLOXL SATL60L SATLZRF 
SATL8CO SATLEXS SATLTZE SATLTZE SATLOMO SATLSZ8 SATL6OW SATLRLW SATLYOA SATLO9O SATLN3M 
SATL447 SATL133 SATLYOA SATLEXS SATL8CO SATLN3M SATLO9O SATLYSR SATLOXL SATLW8S SATLRLW 
SATLO9O SATL0U8 SATLN3M SATL0U8 SATLEXS SATL0U8 SATLFGV SATLTZE SATLOMO SATLEXS SATLFGV SATLEXS SATLFGV SATL60L 
SATL7N8 SATL6OW SATLEXS SATLOMO SATLN3M SATL0U8 SATLTZE SATL6OW SATLOMO SATLOMO SATLO9O SATLYSR SATL60L SATL133 
